raiden_contracts/tests/test_token_network_registry.py

```diff
@@ -144,6 +144,40 @@ def test_create_erc20_token_network(
             == token_network_registry_contract.functions.chain_id().call())
 
 
+def test_create_erc20_token_network_twice_fails(
+        owner,
+        web3,
+        token_network_registry_contract,
+        contracts_manager,
+        custom_token,
+
+):
+    assert token_network_registry_contract.functions.token_to_token_networks(
+        custom_token.address).call() == empty_address
+    #
+    tx_hash = token_network_registry_contract.transact(
+        {'from': owner},
+    ).createERC20TokenNetwork(
+        custom_token.address,
+    )
+    tx_receipt = web3.eth.getTransactionReceipt(tx_hash)
+    event_abi = contracts_manager.get_event_abi(
+        'TokenNetworkRegistry',
+        'TokenNetworkCreated',
+    )
+    event_data = get_event_data(event_abi, tx_receipt['logs'][0])
+    contract_address = event_data['args']['token_network_address']
```

Comment:
We already have a way of checking event arguments - `event_handler` fixture. Why not be consistent? If there is something wrong with `event_handler` consider fixing / refactoring it.